application/gqlgen/post.go

```diff
@@ -260,6 +261,15 @@ func (r *postResolver) Files(ctx context.Context, obj *models.Post) ([]models.Fi
 	return files, nil
 }
 
+// IsEditable response with true if the current user is the creator of the post, and it is not in a locked status.
```

Comment:
comment doesn't seem necessary here and it will need to be maintained should the logic down in `IsEditable` ever changes, I'd recommend losing this one.

Reply:
Good point.